README.md

```diff
@@ -8,14 +8,15 @@ _Note that any changes merged to [api](https://github.com/temporalio/api) will a
 
 To install in your project run:
 ```
-go get -u go.temporal.io/api
+go get go.temporal.io/api
 ```
 
 ## Rebuild
 
 Run `make` once to install all plugins and tools (`protoc` and `go` must be installed manually).
 
-Run `make update-proto` to update submodule and recompile proto files.
+Run `make update-proto` to update the `proto/api` submodule and recompile proto files. The `proto/api-cloud` submodule
+must be updated manually.
```

Comment:
"manually" because you didn't create a `make` target for it or is there another reason?

Reply:
Yes, I didn't want this to be with that other make target because that is run automatically via GH workflow, and we don't want to pull in this repo automatically because it may break in here (there are not checks in that repo that match this repo like the `api` repo has). So updating this submodule is manual, as opposed to a make target auto-run by GH workflow.